torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -412,7 +408,7 @@ def affine_bounding_box(
     # out_bboxes should be of shape [N boxes, 4]
 
     return convert_format_bounding_box(
-        out_bboxes, old_format=features.BoundingBoxFormat.XYXY, new_format=format
+        out_bboxes, old_format=features.BoundingBoxFormat.XYXY, new_format=format, inplace=True
```

Comment:
In kernels where they don't do inplace ops in Bboxes but instead produce new output boxes, we just do an inplace on on the output to convert back their format.